FrontEnd/src/recaptcha

```diff
@@ -0,0 +1 @@
+6Lei8-EUAAAAAHrLbjROeKnbOIROwqLCdOPzHl5z
```

Comment:
I'm guessing this is the secret key for recaptcha. How would you go about securely keeping this key out of source code?

Reply:
This was the 'site key', but I agree, this should be treated as a password and not be in legible text or plain sight.

In retrospect, I should have encrypted it.

Reply:
Well, if it's a public key, it's actually ok for it to be here, your secret key is stored by recaptcha.